<launch>
  <!-- PXRF Handler that exposes wrapper services that allows for services to fake data -->
  <node name="fake_pxrf_handler" pkg="pxrf" type="fake_pxrf_handler.py" output="screen"/>

</launch>